MVC/model.js

```diff
@@ -49,9 +49,31 @@ const fetchArticles = () => {
         })
 };
 
+const fetchCommentsByArticleId = (articleId) => {
+    return db
+        .query(`SELECT
+                comments.comment_id,
+                comments.votes,
+                comments.created_at,
+                comments.author,
+                comments.body,
+                comments.article_id
+            FROM comments
+            WHERE comments.article_id = $1
+            ORDER BY comments.created_at DESC;`, [articleId])
+        .then(({ rows }) => {
+            if (rows.length === 0) {
```

Comment:
if rows.length === 0 this means that there are no comments and this should be a 200. how can you check if the article doesn't exist because if the article exists but there are no comments this is fine.